kai/rpc_server/util.py

```diff
@@ -23,7 +24,7 @@
 def get_prompt(
     template_name: str,
     pb_vars: dict[Any, Any],
-    path_templates: str = PATH_TEMPLATES,
+    path_templates: Path = PATH_TEMPLATES,
```

Comment:
does this get used anywhere?

Reply:
I'm not finding any references to `get_prompt` or `playback_if_demo_mode` anywhere in the project. I can remove this file if we think it's beneficial.

Reply:
its fine for now, but something that we should remember.